README.md

```diff
@@ -26,16 +29,25 @@ git clone https://github.com/coveo-labs/zapier.git
 cd zapier
 npm install
 ```
-Now, you should have the app and all of it's dependencies installed in the directory. If the app failed to update on it's own, run the following commands to
+Now, you should have the app and all of it's dependencies installed in the directory. If the app failed to update or fully install on it's own, run the following commands to
 ensure the app is updated:
 
 ```bash
-sudo npm install zapier-platform-cli
-sudo npm install zapier-platform-core
+sudo npm install -g zapier-platform-cli
+sudo npm install -g zapier-platform-core
```

Comment:
We don't need `zapier-platform-core` as global.
And we don't need to install it explicitly, it will be installed with the `npm install` above.